content/en/blog/2020/mosn-proxy/index.md

```diff
@@ -0,0 +1,124 @@
+
+This is a long-term planning project that takes many years to build and meets the needs of the next five or even ten years, and cooperates to build a team that spans business, SRE, middleware, and infrastructure departments. We must have a network proxy forwarding plane with flexible expansion, high performance, and long-term evolution. Nginx and Envoy have a very long-term capacity accumulation and active community in the field of network agents. We have also borrowed from other excellent open source network agents such as Nginx and Envoy. At the same time, we have enhanced research and development efficiency and flexible expansion. Mesh transformation involves a large number of departments and R & D personnel. We must consider the landing cost of cross-team cooperation. Therefore, we have developed a new network proxy MOSN based on GoLang in the cloud-native scenario. For GoLang's performance, we also did a full investigation and test in the early stage to meet the performance requirements of Ant Group's services.
+
+At the same time, we received a lot of feedback and needs from the end user community. Everyone has the same needs and thoughts. So we combined the actual situation of the community and ourselves to conduct the research and development of MOSN from the perspective of satisfying the community and users. We believe that the open source competition is mainly competition between standards and specifications. We need to make the most suitable implementation choice based on open source standards.
+
+## What is the difference between MOSN and Istio default proxy? What are the advantages of MOSN?
+
+### Differences in language stacks
+
+MOSN is written in GoLang. GoLang has strong guarantees in terms of production efficiency and memory security. At the same time, GoLang has an extensive library ecosystem in the cloud-native era. The performance is acceptable and usable in the service mesh scenario. Therefore, MOSN has a lower intellectual cost for companies and individuals using languages such as GoLang and Java.
+
+### Differentiation of core competence
+
+- MOSN supports a multi-protocol framework, and users can easily access private protocols with a unified routing framework.
+- Multi-process plug-in mechanism, which can easily extend the plug-ins of independent MOSN processes through the plug-in framework, and do some other management, bypass and other functional module extensions.
+- Transport layer national secret algorithm support with Chinese encryption compliance, etc.
+
+### what are the drawbacks of MOSN
+
+- Because MOSN is written in GoLang, it has no better performance than Istio default proxy, but the performance is acceptable and usable in the service mesh scenario.
```

Comment:
```suggestion
- Because MOSN is written in GoLang, it doesn't have as good performance as Istio default proxy, but the performance is acceptable and usable in the service mesh scenario.
```